doc/nrf/protocols/thread/overview/supported_features.rst

```diff
@@ -75,30 +77,30 @@ The available metrics are:
 
 Developers can make use of the OpenThread API to obtain information about the link quality in three ways:
 
-* Single probe: a single-shot MLE command that returns the requested metrics in the reply.
+* Single probe: a single-shot `Mesh Link Establishment (MLE)`_ command that returns the requested metrics in the reply.
 * Forward Tracking Series: the Initiator configures the Subject to start tracking link quality information for every received frame.
   At any point, the Initiator retrieves the averaged values for that series by means of an MLE Data Request.
 * Enhanced-ACK probing: the Initiator configures the Subject to include link metrics information as Information Elements in the Enh-ACKs that are generated in reply to IEEE 802.15.4-2015 frames sent by the Initiator.
 
 Enhanced-ACK probing is the most power efficient method to retrieve link metrics since very little message overhead is required.
-|NCS| provides full Link Metrics support even for the RCP architecture, which is the most technically challenging one since the radio driver must handle the injection of Information Elements on time to match the acknowledgment timing requirements.
+The |NCS| provides full Link Metrics support even for the :ref:`thread_architectures_designs_cp_rcp` architecture, which is the most technically challenging one since the radio driver must handle the injection of Information Elements on time to match the acknowledgment timing requirements.
 The decision on how to interpret the link metrics information to adjust the transmission power is left to the application itself.
 
 Multicast across Thread networks
 ================================
 
-Thread 1.1 border routers have a limitation not to forward multicast traffic with scope greater than realm-local.
+Thread 1.1 border routers have a limitation not to forward multicast traffic with scope greater than `Mesh-Local <IPv6 Addressing Scopes_>`_.
 For certain applications it could be useful to be able to control multicast groups from a host outside the Thread network.
-This is achieved in Thread 1.2 by allowing Thread border routers to forward multicast traffic with scope greater than realm-local in two ways:
+This is achieved in Thread 1.2 by allowing Thread border routers to forward multicast traffic with a scope greater than Mesh-local in two ways:
 
 * From the Thread network to the exterior network: as a configuration option in the border router, for every multicast group.
-* From the exterior network to the Thread network: the Primary Backbone Router (PBBR) would forward only multicast traffic with a destination matching one of the multicast groups registered by Thread devices in its network.
+* From the exterior network to the Thread network: the Primary Backbone Router (PBBR) forwards only multicast traffic with a destination matching one of the multicast groups registered by Thread devices in its network.
```

Comment:
Primary Backbone Router needs an explanation. It's a Backbone Router (BBR) that has been elected to carry out certain tasks, such as (important in this context) forwarding multicasts between segments of heterogeneous network.
BBR is a function of a Border Router. It's not necessarily implemented by all Border Routers, but a Thread device that performs BBR function must be a BR. It's used for operations related to Thread Domains, Device Roaming, and Multicast Forwarding.